projects/1.18/assets/extreme-sound-muffler/extremesoundmuffler/lang/zh_cn.json

```
@@ -0,0 +1,57 @@
+{
+    "key.open_muffler_gui": "打开消音器界面",
+  
+    "inventory.btn": "消音器",
+  
+    "slider.btn.muffler.muffle": "开启消音",
+    "slider.btn.muffler.unmuffle": "关闭消音",
+    "slider.btn.play.play_sound": "播放/暂停音效",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |
没有看懂这条与上两条的关系
```suggestion
    "slider.btn.play.play_sound": "播放/停止音效",
```
原文：**Play/Stop sound**